### DISCUSSION TITLE
Is there a way to "parse HTML codes" into decoded string with this library?

### DISCUSSION THREAD

**Discussion by philhzss** [nlohmann/json#2566] [Q&A] [answered]
Hi all,
Just getting started here so probably a basic question, but I've been trying and researching for a while now and can't get my head around this.

I'm using a combination of cURL and nlohmann::json to make a mini console quiz-app using [this website](https://opentdb.com/api_config.php). It's working quite well, but in my console, some characters (such as quotes, apostrophes etc) are showing like this: `&quot;HTML&quot` and `you&#039;d`

I thought I was doing something wrong with json::parse but I used my browser to view the "raw json" output from the API and it has the same codes. You can view an example [here](https://opentdb.com/api.php?amount=30&category=18&difficulty=easy), and it's generated so you can reload to get a different output if ever there are no weird characters.

I've figured out these are simply HTML codes, but I'm not sure how to convert them back into regular text, could anyone point me in the right direction?

Thank you!

Phil

**Comment by nlohmann** [comment, marked as answer]
I'm afraid this library cannot help you with that.

**Reply by philhzss** [reply]
Thank you very much for taking the time to reply, I ended up parsing the HTML entities myself since I only had a few different ones, so all is good. Thank you for your library and thanks again for taking the time to reply!
